downloader/download.go

```diff
@@ -103,28 +105,30 @@ func (r request) receiveFile(clnt *pool.Pool, objectAddress oid.Address) {
 		return
 	}
 
-	var prm pool.PrmObjectGet
+	var prm client.PrmObjectGet
 	if btoken := bearerToken(r.RequestCtx); btoken != nil {
-		prm.UseBearer(*btoken)
+		prm.WithBearerToken(*btoken)
 	}
 
-	rObj, err := clnt.GetObject(r.appCtx, objectAddress.Container(), objectAddress.Object(), prm)
+	rObj, payloadReader, err := clnt.ObjectGetInit(r.appCtx, objectAddress.Container(), objectAddress.Object(), signer, prm)
```

Comment:
```suggestion
	hdr, payloadReader, err := clnt.ObjectGetInit(r.appCtx, objectAddress.Container(), objectAddress.Object(), signer, prm)
```
to keep name more clear

Reply:
updated